roles/install-calico/tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 - name: Download calico manifest
   get_url:
-    url: https://docs.projectcalico.org/archive/v3.20/manifests/calico.yaml
+    url: https://docs.projectcalico.org/manifests/calico.yaml
```

Comment:
do you really want to always pull the latest stable version or want to lock to any specific version?

Reply:
I think we can use the latest stable as that would also help us identify if there are any issues with calico.

The document [here](https://projectcalico.docs.tigera.io/getting-started/kubernetes/requirements#supported-versions) only talks about supported versions for k8s 1.23 and as these jobs are for 1.24, I feel it is better to run with the latest stable so as to check proper compatibility.